app/javascript/leftSidebar/TagsFollowed.jsx

```diff
@@ -23,6 +23,14 @@ export const TagsFollowed = ({ tags = [] }) => {
             href={`/t/${tag.name}`}
           >
             {`#${tag.name}`}
+            {tag.points < 1 && (
```

Comment:
Is this still an acceptable pattern in modern React and in line with our current practices?

Reply:
Generally I'd suggest using a ternary with `null` as the `&&` approach [can sometimes lead to unexpected consequences](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) (it doesn't in this case though, so please just take this as a general suggestion, and it would definitely be fine and in-keeping with a lot of our other code as-is)

Reply:
Thanks for the link. Since the predicate returns an actual boolean this is one of the safe use-cases, so I'm inclined to leave as-is, as I find it slightly cleaner than the ternary version. Maybe we can standardize one way to do this in the future.

Reply:
Food for thought. Should we even show an anti-follow tag in the sidebar?

Reply:
I wondered about that too. The thing is, once you click on configuring the tags there, the antifollowed tags will show up in the list because, well, that's where you configure the antifollow in the first place. But at one point we also talked about completely removing this tags list from the sidebar, at least when Lisa was still here. I can go either way really but I think you may be right here, maybe removing antifollowed tags from this list is indeed the way to go (and it also means I don't need to mess around with CSS which is probably better for everyone involved).

Reply:
I updated the code to no longer display anti-followed tags in the left sidebar and also to use a ternary.